src/App.js

```diff
@@ -1,14 +1,14 @@
 import { useState } from 'react';
 
-function Square({ value, onSquareClick }) {
-  return <button className="square" onClick={onSquareClick}>{value}</button>;
+function Square({ value, onSquareClick, className }) {
+  return <button className={`square ${className}`} onClick={onSquareClick} >{value}</button>;
 }
 
 function Sorter({ description, onSorterClick}) {
   return <button className="sorter" onClick={onSorterClick}>{description}</button>
 }
 
-function Board({ xIsNext, squares, onPlay }) {
+function Board({ xIsNext, squares, onPlay, currentMove }) {
   function handleClick(i) {
```

Comment:
Change the argument's name to more descriptive.

Reply:
Done.